app/javascript/packages/verify-flow/context/flow-context.tsx

```diff
@@ -9,7 +9,7 @@ export interface FlowContextValue {
   /**
    * URL to in-person proofing alternative flow, if enabled.
    */
-  inPersonURL: string | null;
+  inPersonURL?: string | null;
```

Comment:
Will this do?
```suggestion
  inPersonURL?: string;
```

Reply:
Based on usages, I expect not. `inPersonURL?: string` really means that the key should be absent - i.e. not `null` nor even set to `undefined` though that is often still safe.

Reply:
I'll check to see if we're setting it `null` anywhere currently, and if not, will try to simplify to `inPersonURL?: string`.

Reply:
Needed a couple updates, but simpler overall. Updated in b4493c950.